### DISCUSSION TITLE
Some way to fix position

### DISCUSSION THREAD

**Discussion by skuzzymiglet** [qutebrowser/qutebrowser#6020] [Ideas] [answered]
It would be nice if there was a way to stop qutebrowser from scrolling on a page (e.g. resizes, searches). When searching through docs and juggling windows, loosing position is a bit annoying.

Something like:

```
:fix-position
:unfix-position
:toggle-fix-position
```

**EDIT** probably `position-fix`, `position-unfix` and `position-fix-toggle` for consistency, I guess

**Comment by The-Compiler** [comment, marked as answer]
I don't think there is a way for qutebrowser to prevent the underlying QtWebEngine/Chromium from scrolling around.

However, for some actions (like searching), qutebrowser remembers the last scroll position in the `'` mark, so you can use `''` (like in vim) to jump back.
